serverlessworkflow/modules/ROOT/pages/eventing/consume-producing-events-with-kafka.adoc

```diff
@@ -33,53 +26,59 @@ If using Maven, this means including following snippet in pom.xml:
-
-You can change the default incoming and outgoing topic name as follows:
+To change the default incoming and outgoing topic names, you can use the following properties:
 
+.Properties to change default incoming and outgoing topic names
 [source,properties]
 ----
 kogito.addon.messaging.incoming.defaultName=<default channel name>
 kogito.addon.messaging.outgoing.defaultName=<default channel name>
 ----
 
-This is the full list of link:{kafka_connector_properties_url}[properties] you can use for a channel when using Kafka connector.
+The following properties are mandatory that you need to set for each channel:
 
-There are two mandatory properties that you need to setup for every channel defined:
+* `connector`: This property needs to be set to `smallrye-kafka`.
+* Depending on whether the channel is incoming or outgoing:
+** `value.deserializer`: This property is used for incoming channels. Unless you have specific marshaling requirements, you must set this property to `org.apache.kafka.common.serialization.ByteArrayDeserializer` or `org.apache.kafka.common.serialization.StringDeserializer`.
+** `value.serializer`: This property is used for outgoing channels. Unless you have specific marshaling requirements, you must set this property to `org.apache.kafka.common.serialization.ByteArraySerializer` or `org.apache.kafka.common.serialization.StringSerializer`.
 
-* `connector`, which has to be set to `smallrye-kafka`
-* Depending on the channel being input or output: 
-** if incoming, `value.deserializer`. Unless you have specific marshaling needs, you should set it to `org.apache.kafka.common.serialization.ByteArrayDeserializer` or `org.apache.kafka.common.serialization.StringDeserializer`
-** if outgoing, `value.serializer`. Unless you have specific marshaling needs, you should set it to `org.apache.kafka.common.serialization.ByteArraySerializer` or `org.apache.kafka.common.serialization.StringSerializer`
+[TIP]
+====
+`topic` is another relevant but optional property that needs to be set for a channel. The `topic` property must contain the Apache Kafka topic name to be used for a channel. If `topic` property is not set, then the channel name is used as topic name. 
+====
 
-Another relevant, although optional, property is the `topic`. If present, it should contain the Kafka topic name to be used for that channel. If missing, the channel name will be used as topic name. 
+For more information about the properties that you can use for a channel when using Apache Kafka connector, see the link:{kafka_connector_properties_url}[properties] list.
 
-=== Channel Mapping Examples
+[[ref-example-smallrye-channel-mapping]]
+=== Examples of Smallrye channel mapping
 
-Let's illustrate previous explanation by reviewing the properties configured in several examples. 
+This section describes the examples of channel mapping by reviewing the properties that are configured in the previous section. 
```

Comment:
"This section describes the examples of channel mapping by reviewing the properties that are configured in the previous section. "
should be
"This section describes some examples of channel mapping, using the properties mentioned in the previous section"

Reply:
Done